lib/Chat/Parser/SystemMessage.php

```diff
@@ -732,6 +735,21 @@ protected function getFileFromShare(?Participant $participant, string $shareId):
 			'preview-available' => $this->previewManager->isAvailable($node) ? 'yes' : 'no',
 		];
 
+		// If a preview is available, check if we can get the dimension of the 
```

Comment:
Of the what? Don't leave me hanging 🤔

Reply:
> Of the what? Don't leave me hanging 🤔

Yea, saw it after pushing and added a task above to change it later 😅